scripts/whitelist.py

```diff
@@ -41,6 +41,7 @@
 _.convert  # unused method (aea/cli/utils/click_utils.py:104)
 _.convert  # unused method (aea/cli/utils/click_utils.py:129)
 _.formatter  # unused attribute (aea/cli/utils/loggers.py:92)
+is_item_present_unified  # unused function (aea/cli/utils/package_utils:394)
```

Comment:
I added the method just for completeness with the analogous `get_package_path_unified`, but I didn't use it. That's why it is here; we could remove it anyway.